### PR TITLE
Export hidden and more 126890643

### PR DESCRIPTION
**Pull request by jjdelc**

### CODE REVIEW

File: source/includes/_endpoint-dataset.md [hunk 1]
```diff
@@ -805,6 +816,7 @@ Some format-specific properties and options:
 Format    | Attribute        | Description                                                      | Example
 --------- | ---------------- | ---------------------------------------------------------------- | --------------------------
 csv       | use_category_ids | instead of category names export the fields as their numeric ids | {"use_category_ids": true}
+csv       | include_personal | Will include the user's personal variables on the exported file  | {"include_personal": true}
```
Comment (nealrichardson) [inline source/includes/_endpoint-dataset.md]: This is only for CSV? Why would that be?

File: source/includes/_endpoint-dataset.md [hunk 2]
```diff
@@ -795,8 +806,8 @@ See ["Expressions"](#expressions) for more on Crunch expressions.
 The following rules apply for all formats:
 
 * The dataset's exclusion filter will be applied; however, any of the user's personal "applied filters" are not, unless they are explicitly included in the request.
-* Hidden/discarded variables are not exported.
-* Personal (private) variables are not exported.
+* Hidden/discarded variables are not exported unless editors use a `where` clause, then it will be evaluated over all non hidden variables.
```
Comment (nealrichardson) [inline source/includes/_endpoint-dataset.md:809]: This is not clear and sounds self-contradicting. I think you mean that you won't get hidden variables unless you request them explicitly (or with *), and even then only if you have edit privileges on the dataset.

File: source/includes/_endpoint-dataset.md [hunk 3]
```diff
@@ -750,27 +750,38 @@ GET returns a Shoji View of available dataset export formats.
...
 POST `/api/datasets/f2364cc66e604d63a3be3e8811fc902f/export/spss/` HTTP/1.1
 ```
 ```json
-{"where":{"function": "identify",
-          "args":[{"id":["https://beta.crunch.io/api/datasets/f2364cc66e604d63a3be3e8811fc902f/variables/000000/",
-                         "https://beta.crunch.io/api/datasets/f2364cc66e604d63a3be3e8811fc902f/variables/000001/",
-                         "https://beta.crunch.io/api/datasets/f2364cc66e604d63a3be3e8811fc902f/variables/000002/"]}]}}
+    {
+      "where": {
+        "function": "identify",
```
Comment (nealrichardson) [inline source/includes/_endpoint-dataset.md]: Haven't you deprecated "identify"? This example should reflect best practice. And if there are features of this select "map" API that are non obvious (hint: there are :), they should be documented. Perhaps document them in the Crunch Expression section and link back to them here (and in the other APIs where we accept frames defined that way).

File: source/includes/_endpoint-dataset.md (hunk 1 above)
Comment (jjdelc) [reply source/includes/_endpoint-dataset.md]: Right, that works for all. This happens independent on the format for variable picking. Will change to "all"

**Pull request merged**